### PR TITLE
Refactor/harcoded strings

### PR DESCRIPTION
**Pull request by ChrisAusema**
The Harcoded Strings are now called from withing the string.xml file

### CODE REVIEW

File: Hestia/app/src/main/java/hestia/UI/dialogs/AddDeviceDialog.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public void setFragmentManager(FragmentManager fragmentManager) {
 
     @Override
     String buildTitle() {
-        return "new Add device";
+        return getContext().getString(R.string.addDeviceTitle);
```
Comment (StefanEvanghelides) [inline Hestia/app/src/main/java/hestia/UI/dialogs/AddDeviceDialog.java:53]: This quite a strange name for a dialog. I believe that simply "Add device" should do.

File: Hestia/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -49,4 +49,16 @@
...
     <string name="userSet">Username set to: </string>
     <string name="userNotSet">Username not changed (length under 5)</string>
+    <string name="deviceEndpoint">devices/</string>
+    <string name="activatorEndpoint">/activators/</string>
+    <string name="pluginEndpoint">plugins/</string>
+    <string name="loginEndpoint">login/</string>
+    <string name="ioExceptionMessage">Could not connect to the server</string>
+    <string name="enterIpHint">Enter ip here</string>
+    <string name="credentialsCorrectlyChanged">Credentials successfully changed</string>
+    <string name="addDeviceTitle">new Add device</string>
```
Comment (StefanEvanghelides) [inline Hestia/app/src/main/res/values/strings.xml]: As stated earlier, "new Add device" is quite a strange name for a dialog. Simply "Add device" should do in my opinion

**Comment by StefanEvanghelides** [comment]
Good changes. Well done Chris.

File: Hestia/app/src/main/java/hestia/UI/dialogs/AddDeviceDialog.java (hunk 1 above)
Comment (LarsHoldijk) [reply Hestia/app/src/main/java/hestia/UI/dialogs/AddDeviceDialog.java:53]: This is my mistake. I changed the name to " new Add device" to see whether or not the dialog actually changed. Forgot to change it back.

**Review by LarsHoldijk — CHANGES_REQUESTED** [review]
To be honest I dont think having all string in one big file is helpfull. It is quite easy to make seperate string files for every class.

**Pull request closed without merging**